templates/artist/artist_application.html

```diff
@@ -1,11 +1,11 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h3 class="extra-padding">Artist Application</h3>
+    <h3 class="extra-padding">Create An Artist Account</h3>
```

Comment:
Should this change also be in https://github.com/RevolutionTech/perdiem-django/pull/363? After you apply that change to the other PR I can revert this file.